src/components/protocol_handler/src/protocol_handler_impl.cc

```diff
@@ -103,6 +103,8 @@ ProtocolHandlerImpl::ProtocolHandlerImpl(
       get_settings().maximum_audio_payload_size());
   protocol_header_validator_.set_max_video_payload_size(
       get_settings().maximum_video_payload_size());
+  protocol_header_validator_.set_max_protocol_version_supported(
+      get_settings().max_supported_protocol_version());
```

Comment:
@SKobziev if i understand correctly, `max_supported_protocol_version()` returns `uint16_t`, whereas `set_max_protocol_version_supported` expects `uint8_t`. This can potentially cause problems.

Reply:
It's corrected.